gpt_playground/settings.py

```diff
@@ -548,6 +548,7 @@
     "account_reset_password_from_key",
     "teams:signup_after_invite",
     "account_login",
+    "single_team:delete_team",
```

Comment:
Why was this necessary?

Reply:
See https://github.com/dimagi/open-chat-studio/pull/1169#discussion_r1947136972